### DISCUSSION TITLE
need help on SQL query execution Reason: SQL Error [53300]

### DISCUSSION THREAD

**Discussion by Tutuldot** [supabase/supabase#13461] [Questions] [answered]
Hi,

Our team is new to Postgres and We are currently developing ETL scripts using dbeaver and python. After an hour of session we always encounter this issue

error:

supabase Error occurred during SQL query execution Reason: SQL Error [53300]: FATAL: remaining connection slots are reserved for non-replication superuser connections

how to resolve this?

Thank you

Anthony

**Comment by GaryAustin1** [comment, marked as answer]
You probably need to look at using the PGbouncer port.
https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool

**Reply by Tutuldot** [reply]
thanks
